Formula/qtrn.rb

```diff
@@ -0,0 +1,26 @@
+class Qtrn < Formula
+  desc "Cli tool to streamline financial markets data analysis"
+  homepage "https://github.com/FlashBoys/qtrn"
+  url "https://github.com/FlashBoys/qtrn/archive/v0.3.tar.gz"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    dir = buildpath/"src/github.com/FlashBoys/qtrn"
+    dir.install Dir["*"]
+    cd dir do
+      system "glide", "install"
+      system "go", "build", "-o", "build/qtrn",
+        "-ldflags",
+        "'-X=github.com/FlashBoys/qtrn/version.Version=#{version}'"
+      bin.install "build/qtrn"
```

Comment:
No, there’s no way. Ultimately it’s the maintainers’ decision to accept or reject new formulae but `brew audit` automates the basic checks. In your case this is a self-submission of a relatively new and not well-known project (3 stars on GitHub) so it doesn’t meet the [requirements](https://docs.brew.sh/Acceptable-Formulae.html#niche-or-self-submitted-stuff) to get in Homebrew core. You can, however, maintain a formula [in your own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap.html) and re-submit it here once it gets more popular.